Update SecurityAdvisoryHandling.md based on feedback (#3709)
openenclave/openenclave: docs/Community/SecurityAdvisoryHandling.md

The CVD includes a couple of policies such as:

- acknowledgment of vulnerability reports within 24 hours (via MSRC)
- supporting public disclosure within 90 days as the baseline and negotiating the date with vulnerability reporters and necessary parties, including the engineering teams and affected vendors.

While the primary goal of CVD is to avoid disclosure before a fix is released, the OE SDK security reviewers will also work to
accommodate the reporter's preferences on the timing for disclosing and fixing the vulnerability. For example, if a researcher
believes that releasing the fix far ahead of their disclosure publication or presentation diminishes the impact of their work, the
security reviewers will work with them to determine the best course of action, such as releasing the fix on the same day as their
planned disclosure event.

> - [ ] TODO: Establish a process and acceptance criteria for registering with OE SDK as a _Trusted Stakeholder_.

> >   - A Trusted Stakeholder may not be a committer or maintainer of the SDK but has a vested interest in the security of the OE SDK.

Reviewer: ```suggestion
>   - A Trusted Stakeholder need not be a committer or maintainer of the SDK but can be anyone with a vested interest in the security of the OE SDK.
```

Author: Updated as suggested.
